tests/util.py

```diff
@@ -6,6 +6,9 @@
 from impy import models as im
 from impy.common import MCRun
 import pytest
+import platform
+
+from impy.models.urqmd import UrQMD34
```

Comment:
util should not import from impy.models to avoid potential circular dependencies. After merging the Pythia8 branch, you can do `if obj.pyname == "UrQMD34"` in line 64, which works without importing UrQMD34.

Reply:
The change was accidental. It should be `im.UrQMD34` so it is changed to that.